osu.Game.Rulesets.Osu/Edit/TransformToolboxGroup.cs

```diff
@@ -53,7 +53,7 @@ protected override void LoadComplete()
 
             // bindings to `Enabled` on the buttons are decoupled on purpose
             // due to the weird `OsuButton` behaviour of resetting `Enabled` to `false` when `Action` is set.
-            canRotate.BindTo(RotationHandler.CanRotate);
+            canRotate.BindTo(RotationHandler.CanRotatePlayfieldOrigin);
```

Comment:
This will aggregate the two bindables in next commit. Wont push now to reduce CI load

Reply:
Pushed